pin/gc/gc.go

```diff
@@ -100,10 +105,5 @@ func ColoredSet(pn pin.Pinner, ds dag.DAGService) (key.KeySet, error) {
 		gcs.Add(k)
 	}
 
-	err = Descendants(ds, gcs, pn.InternalPins())
```

Comment:
no, because the InternalPins represent the objects stored in the state dag, and the `ColoredSet` function returns objects pinned in the data dag. So we move that elsewhere